### PR TITLE
Add the integration tutorials

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

### CODE REVIEW

File: pages/learn/integration/integration-tutorials/index.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
```
Comment (dulajdilshan) [inline pages/learn/integration/integration-tutorials/index.js:2]: 2023

Comment (dulajdilshan) [reply pages/learn/integration/integration-tutorials/index.js:2]: https://github.com/ballerina-platform/ballerina-dev-website/pull/7941

File: pages/learn/integration/integration-tutorials/[slug].js [hunk 2]
```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
```
Comment (mindula) [inline pages/learn/integration/integration-tutorials/[slug].js:2]:
```suggestion
 * Copyright (c) 2023, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
```

File: pages/learn/integration/integration-tutorials/index.js (hunk 1 above)
Comment (mindula) [inline pages/learn/integration/integration-tutorials/index.js:2]:
```suggestion
 * Copyright (c) 2023, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
```

**Review by mindula — DISMISSED** [review]

File: pages/learn/integration/integration-tutorials/index.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        <Col sm={12}>
+          <Row className="pageHeader pageContentRow llanding">
+
+            <Col xs={12}>
+              <Container>
+                <h1>Integration tutorials</h1>
+              </Container>
+            </Col>
+
+
```
Comment (dulajdilshan) [inline pages/learn/integration/integration-tutorials/index.js:124]: Two new lines. I'm not sure why we do this. I have seen this other places too (minor comment)

@sm1990/ @keizer619  Any Idea?

File: pages/learn/integration/integration-tutorials/index.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+                    tags={["Email", "SMTP Client", "HTTP Client", "HTTP Service", "REST API", "Integration"]}
+                    icon={false}
+                  />
+
+                  {/* <SampleList
+                    name="File processing"
+                    description="File processing"
+                    tags={["File Processing", "Persistence", "Integration"]}
+                    icon={false}
+                  /> */}
```
Comment (dulajdilshan) [inline pages/learn/integration/integration-tutorials/index.js:183]: (minor comment) Is it okay to keep this commented one?

@MaryamZi?

**Review by dulajdilshan — DISMISSED** [review]

**Review by dulajdilshan — APPROVED** [review]

**Review by dulajdilshan — APPROVED** [review]

**Pull request merged**